### PR TITLE
teleprompter: init at 2.3.4

### PR DESCRIPTION
**Pull request by Scriptkiddi**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/telepromter/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchurl, electron, makeDesktopItem, makeWrapper, nodePackages, autoPatchelfHook}:
+
+stdenv.mkDerivation rec {
+  pname = "telepromter";
+  version = "2.3.4";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/teleprompter-imaginary-films/files/${version}/imaginary-teleprompter-${version}-64bit.tar.gz";
```
Comment (ajs124) [inline pkgs/applications/misc/telepromter/default.nix]: Use `mirror://sourceforge/teleprompter-imaginary-films/${version}/imaginary-${pname}-${version}-64bit.tar.gz` instead.

File: pkgs/applications/misc/telepromter/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+     name = "telepromter";
+     exec = "telepromter";
+     type = "Application";
+     desktopName = "Telepromter";
+  };
+
+  meta = {
+    description = "The most complete, free, teleprompter app on the web";
+    license = with lib.licenses; [ gpl3 ];
+    homepage = "https://www.precice.org://github.com/ImaginarySense/Teleprompter-Core";
```
Comment (ajs124) [inline pkgs/applications/misc/telepromter/default.nix]: What?

**Review by ajs124 — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/telepromter/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  '';
+
+  desktopItem = makeDesktopItem {
+     name = "telepromter";
+     exec = "telepromter";
+     type = "Application";
+     desktopName = "Telepromter";
+  };
+
+  meta = {
```
Comment (mmahut) [inline pkgs/applications/misc/telepromter/default.nix]: Consider using ```with stdenv.lib```.

**Comment by Scriptkiddi** [comment]
@mmahut fixed

File: pkgs/applications/misc/teleprompter/default.nix [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+     name = "teleprompter";
+     exec = "teleprompter";
+     type = "Application";
+     desktopName = "Teleprompter";
+  };
+
+  meta = with lib; {
+    description = "The most complete, free, teleprompter app on the web";
+    license = [ licenses.gpl3 ];
+    homepage = "https://www.precice.org://github.com/ImaginarySense/Teleprompter-Core";
```
Comment (ajs124) [inline pkgs/applications/misc/teleprompter/default.nix]: bruh

**Review by Lassulus — APPROVED** [review]
tested with nix-review

**Pull request merged**